src/commands/ProjectCommand.ts

```diff
@@ -0,0 +1,60 @@
+import Command from "../abstracts/Command";
+import {Message, MessageEmbed} from "discord.js";
+import {readFile} from "fs";
+
+export default class ProjectCommand extends Command {
+	private readonly fileDirectory = "./assets/projects.json";
+	constructor() {
+		super(
+			"project",
+			"Returns a random project idea based on given parameters.",
+		);
+	}
+
+	async run(message: Message, args: string[]): Promise<void> {
+		const embed = new MessageEmbed();
+		const query = args.map(arg => arg.toLowerCase()).filter(arg => arg.trim().length > 0);
+
+		if (args.length === 0) {
+			embed.setTitle("Projects")
+				.setColor("#add8e6")
+				.setDescription("Please provide arguments on the projects command.");
```

Comment:
Project convention isn't to chain embed creation methods, can this be changed please.